ui/src/app/utils/index.tsx

```diff
@@ -277,6 +277,16 @@ export const withCurrentWorkspace = () => {
   return connectBehaviorSubject(currentWorkspaceStore, 'workspace');
 };
 
+// HOC that provides a 'workspace' prop with current WorkspaceData
```

Comment:
Stale doc, here and on the other changes in this file

Reply:
Good catch, fixed